users/zyber/zyber.c

```diff
@@ -0,0 +1,160 @@
+#include "zyber.h"
+
+
+__attribute__((weak)) bool process_record_keymap(uint16_t keycode, keyrecord_t *record) { return true; }
+
+uint16_t key_timer;
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	if (record->event.pressed) {
+		key_timer = timer_read();
+	}
+
+	if (!process_caps_word(keycode, record)) {
+		return false;
+	}
+
+	switch(keycode) {
+		case C_BLK:
+			if (record->event.pressed) {
+				send_string("```" SS_DELAY(80) SS_LCMD("v") SS_DELAY(80) "```");
+			}
+			break;
+
+		case CTRL_C_UP:
+			if (record->event.pressed) {
+				register_mods(MOD_BIT(KC_LCTRL));
+			} else {
+				unregister_mods(MOD_BIT(KC_LCTRL));
+				if (timer_elapsed(key_timer) < TAPPING_TERM) {
+					tap_code16(C(KC_UP));
+				}
+			}
+			return false;
+			break;
+
+		case L1_EXPL:
+			if (record->event.pressed) {
+				layer_on(1);
+			} else {
+				layer_off(1);
+				if (timer_elapsed(key_timer) < TAPPING_TERM) {
+					tap_code16(C(KC_UP));
+				}
+			}
+			return false;
+			break;
+
+		case ROPT_SRH:
+			if (record->event.pressed) {
+				register_mods(MOD_BIT(KC_ROPT));
+			} else {
+				unregister_mods(MOD_BIT(KC_ROPT));
+				if (timer_elapsed(key_timer) < TAPPING_TERM) {
+					tap_code16(G(KC_SPC));
+				}
+			}
+			return false;
+			break;
+	}
+
+	return process_record_keymap(keycode, record);
+}
+
+/* Screenshoot */
+void dance_SSHT_finished(qk_tap_dance_state_t *state, void *user_data) {
+	if (state->count == 2) {
+		register_code(KC_LGUI);
+		register_code(KC_LCTRL);
+		register_code(KC_LSHIFT);
+		register_code(KC_4);
+		unregister_code(KC_LGUI);
+		unregister_code(KC_LCTRL);
+		unregister_code(KC_LSHIFT);
+		unregister_code(KC_4);
```

Comment:
```suggestion
                tap_code16(C(S(G(KC_4))));
```

Reply:
OMG, thank you I didn't know I could do this :)